### ISSUE TITLE
About Python 3.11.2 Runtime errors

### ISSUE THREAD

**Issue by friendbear** [monkeymademe/picamera2-WebUI#60]
**Describe the bug**

Runtime error in app.py in the following runtime environment.
Please provide support to address the issue.

<img width="533" alt="image" src="https://github.com/user-attachments/assets/6a3fab71-75e3-448f-bb3a-a6ea53fa26e9">

* Python 3.11.2

**To Reproduce**

1. git clone
1. app.py running on rasberry pi

**Expected behavior**

**Screenshots**

```bash
python3 GitHub/picamera2-WebUI/app.py
[0:18:00.132399226] [5330]  INFO Camera camera_manager.cpp:313 libcamera v0.3.0+65-6ddd79b5
[0:18:00.289671828] [5330]  INFO Camera camera_manager.cpp:313 libcamera v0.3.0+65-6ddd79b5

Camera Config:
{'controls': {'LensPosition': 1.0, 'ExposureValue': 0.0, 'ExposureTime': 220417486, 'AwbMode': 0, 'AfSpeed': 0, 'AfRange': 0, 'AeMeteringMode': 0, 'AeFlickerPeriod': 1000000, 'AeFlickerMode': 0, 'AeExposureMode': 0, 'AeEnable': True, 'AwbEnable': True, 'AeConstraintMode': 0, 'Brightness': 0.0, 'Contrast': 1.0, 'Saturation': 1.0, 'Sharpness': 1.0, 'AfMode': 0, 'ScalerCrop': [0, 0, 4608, 2592]}, 'rotation': {'hflip': 0, 'vflip': 0}, 'sensor-mode': 0, 'capture-settings': {'Resize': False, 'makeRaw': False, 'Resolution': 0, 'available-resolutions': [[4608, 2592], [2304, 1296], [1920, 1080], [1280, 720], [640, 360]]}}


Camera Settings:
{'Resize': False, 'makeRaw': False, 'Resolution': 0, 'available-resolutions': [[4608, 2592], [2304, 1296], [1920, 1080], [1280, 720], [640, 360]]}


Camera Set Resolution:
[4608, 2592]

Traceback (most recent call last):
  File "/home/pi/GitHub/picamera2-WebUI/app.py", line 54, in <module>
    video_config = picam2.create_video_configuration(main={'size':resolution}, sensor={'output_size': mode['size'], 'bit_depth': mode['bit_depth']})
                                                                                                      ~~~~^^^^^^^^
KeyError: 'size'
```

**Desktop (please complete the following information):**
 - OS: [macOS 14.6]
 - Browser Firefox

**Additional context**

**Comment by monkeymademe** [comment]
I have released v1.0.1 https://github.com/monkeymademe/picamera2-WebUI/releases/tag/v1.0.1 which might already resolve this issue

**Comment by friendbear** [comment]
Thank you.
